warehouse/ingest-core/src/test/java/datawave/iterators/SortedMultiMapIterator.java

```diff
@@ -115,9 +114,4 @@ public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean i
     public void init(SortedKeyValueIterator<Key,Value> source, Map<String,String> options, IteratorEnvironment env) throws IOException {
         throw new UnsupportedOperationException();
     }
-
-    @Override
-    public void setInterruptFlag(AtomicBoolean flag) {
-        this.interruptFlag = flag;
```

Comment:
I believe we can now remove this flag as well and places where we use it in this class.